satellite/console/service.go

```diff
@@ -178,23 +179,36 @@ func (s *Service) CreateUser(ctx context.Context, user CreateUser, tokenSecret R
 	return u, nil
 }
 
-// AddNewPaymentMethod adds new payment method for project
-func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken string, isDefault bool, projectID uuid.UUID) (payment *ProjectPayment, err error) {
+// AddNewProjectPaymentMethod adds new payment method for project
+func (s *Service) AddNewProjectPaymentMethod(ctx context.Context, paymentMethodToken string, isDefault bool, projectID uuid.UUID) (payment *ProjectPayment, err error) {
 	defer mon.Task()(&ctx)(&err)
-
 	authorization, err := GetAuth(ctx)
 	if err != nil {
 		return nil, err
 	}
 
+	var customerID []byte
 	userPayments, err := s.store.UserPayments().Get(ctx, authorization.User.ID)
 	if err != nil {
-		return nil, err
+		if err != sql.ErrNoRows {
```

Comment:
Service shouldn't know anything about sql. The particular database is an implementation detail.